testsuite-native-image-client/pom.xml

```diff
@@ -78,7 +78,7 @@
           <skip>${skipNativeImageTestsuite}</skip>
           <imageName>${project.artifactId}</imageName>
           <mainClass>io.netty.testsuite.svm.client.DnsNativeClient</mainClass>
-          <buildArgs>--report-unsupported-elements-at-runtime --allow-incomplete-classpath --no-fallback --initialize-at-build-time=io.netty -H:ReflectionConfigurationResources=reflection-config.json</buildArgs>
+          <buildArgs>--report-unsupported-elements-at-runtime --allow-incomplete-classpath --no-fallback -H:ReflectionConfigurationResources=reflection-config.json</buildArgs>
```

Comment:
The file `reflection-config.json` should be renamed to `reflect-config.json` (and maybe moved in the correct directory), so that the `-H:ReflectionConfigurationResources` argument is not necessary at all because the file is found automatically.

Or is that the `reflection-config.json` file that shows up as "deleted" in the PR? Then we don't need the argument either.

Reply:
Please check the whole Netty code base for `-H:` - there should not be any need to have such an option in any `native-image.properties` file.

Reply:
This is the only `-H:` that is left.